starters/gatsby-wordpress-starter/gatsby-config.js

```diff
@@ -1,14 +1,25 @@
 const path = require("path")
+const fs = require("fs")
+
 require("dotenv").config({
-  path: path.resolve(process.cwd(), ".env.local"),
+  path: path.resolve(process.cwd(), ".env.development.local"),
 })
 
 // Use URL from .env if it exists, otherwise fall back on the
 // Pantheon CMS endpoint
+
+if (
+  process.env.WPGRAPHQL_URL === undefined &&
+  process.env.PANTHEON_CMS_ENDPOINT === undefined
+) {
+  throw new Error(
+    "Please set critical environment variables PANTHEON_CMS_ENDPOINT or WPGRAPHQL_URL."
+  )
+}
```

Comment:
The check for the env var looks good, but the error message still doesn't pass the AC on the ticket:

- the error message should suggest ways to fix the problem by checking the NODE_ENV and suggesting an appropriate fix for each 

  - set a .env.development.local with the critical envs for NODE_ENV=development

  - link a backend or set the critical env var in the dashboard for NODE_ENV=production